internal/services/webhooks/webhooks.go

```diff
@@ -337,6 +342,11 @@ func (c *WebhooksController) run(tenantId string, webhookWorker *dbsqlc.WebhookW
 			case <-ctx.Done():
 				return
 			case <-ticker.C:
+
+				if !c.registeredWorkerIds[sqlchelpers.UUIDToStr(webhookWorker.ID)] {
```

Comment:
i'm not sure if i need to cancel() here... i dont think so because we're returning